src/main/com/example012/v1/Company.java

```diff
@@ -0,0 +1,35 @@
+package main.com.example012.v1;
+
+public enum Company {
+
+    SPORTKA("Sportka", 49, 6),
+    STASTNYCH10("Štastných 10", 80, 10),
+    EUROJACKPOT("Eurojackpot", 50, 5);
+
+
+    final String company;
```

Comment:
company není dobrý název fieldu ... t ze to je company uz vime z nazvu teho enumu ... dal bych tam name ... pak to bude Company.name